web/src/layout/Header/Logo.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled, { Theme, useTheme } from "styled-components";
+import KlerosCourtLogo from "svgs/header/kleros-court.svg";
+
+const arbitratorType = process.env.REACT_APP_ARBITRATOR_TYPE;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 16px;
+`;
+
+const StyledLink = styled(Link)`
+  min-height: 48px;
+`;
+
+const BadgeContainer = styled.div<{ backgroundColor: string }>`
+  transform: skewX(-15deg);
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  border-radius: 3px;
+  padding: 1px 8px;
+  height: fit-content;
+`;
+
+const BadgeText = styled.label`
+  color: #220050;
```

Comment:
I think it's better if we don't hardcode the color, either get it from the dark theme white background or add it as `logoBadgeText` (or a better name 😅 ) with the other two colors that have been added.